### PR TITLE
fix: fetch if holder_of exists

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Context

- [x] Fix query error on drops page. Fetch if holder_of exists

screenshot-1710601166893

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)

### CODE REVIEW

File: composables/drop/useHolderOfCollection.ts [hunk 1]
```diff
@@ -74,15 +73,9 @@ export default function useHolderOfCollection() {
           id: drop.holder_of,
           account: accountId.value,
         },
-      }).then((res) => (holderOfCollectionData.value = res.data.value)),
-    {
-      watch: [
-        accountId,
-        () => runtimeMintCount,
```
Comment (Jarsen136) [inline composables/drop/useHolderOfCollection.ts:81]: How do we deal with the update from this one `runtimeMintCount`? It's not watched then.

Comment (preschian) [reply composables/drop/useHolderOfCollection.ts:81]: updated

**Comment by yangwao** [comment]
oh, this is the root of the Stellate alerts we've got?

**Comment by preschian** [comment]
> oh, this is the root of the Stellate alerts we've got?

one of them. some of the errors coming from localhost (safe to ignore). another one not able to reproduce it yet

**Comment by yangwao** [comment]
Thanks for great find and less Stellate alerts!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 9.86 USD/DOT ~ 5.071 $DOT
🧗 [1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p](https://polkadot.subscan.io/account/1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)
🔗 [0xfd24ed615a2ca9d7dd7ee3c312d311ab04e95669789343ece7fe2eb5ae5d5516](https://polkadot.subscan.io/extrinsic/0xfd24ed615a2ca9d7dd7ee3c312d311ab04e95669789343ece7fe2eb5ae5d5516)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**